### ISSUE TITLE
Kotlin script file reported as empty (false positive)

### ISSUE THREAD

**Issue by paul-dingemans** [pinterest/ktlint#2415]
Given that code below is saved as Kotlin script `Bar.kts`
```
import java.io.FileInputStream
```

Ktlint falsely reports this file as being empty:
```
src/main/kotlin/Bar.kts:1:1: File 'Bar.kts' should not be empty (standard:no-empty-file)
src/main/kotlin/Bar.kts:1:1: Unused import (standard:no-unused-imports)
```

**Comment by paul-dingemans** [comment]
The package statement and the imports are ignored when checking whether a file is empty. The message is a bit misleading but does not seem to cause a lot of confusion.
